db/migrations/20190911160416_drop_title_from_movies.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+exports.up = (Knex) => {
+  return Knex.schema.table('movies', (table) => {
+    table.dropColumn('title');
+  })
+  .then(() => {
+    return Knex.raw('ALTER TABLE movies ADD CONSTRAINT movies_name_not_null CHECK (name IS NOT NULL) NOT VALID')
```

Comment:
Notice how we reference the `movies` table and the `movies_name_not_null` constraint multiple times in this file. One way to reduce error is to set constants to these values and reference them to reduce potential error (e.g. we might have a typo on the constraint when typing it again).

So you could set 
```
const CONSTRAINT = 'movies_name_not_null`;
const TABLE      = 'movies';
```

at the top of the file and then reference them as string literals with backticks, .e.g. \`ALTER TABLE ${movies}...\`

Reply:
good to know!